lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -428,6 +428,32 @@
       - powershell.exe -ExecutionPolicy Unrestricted -File C:\Windows\Temp\ConfigureRemotingForAnsible.ps1 -ForceNewSSLCert -EnableCredSSP
   delegate_to: localhost
 
+- name:  Clone a virtual machine from Linux template and customize
+  vmware_guest:
+    hostname: "{{ vcenter_server }}"
+    username: "{{ vcenter_user }}"
+    password: "{{ vcenter_password }}"
+    validate_certs: False
```

Comment:
We standardized to using yes/no booleans in documentation (mostly because it is easier to read and reflects programming less).